core/domain/exp_services.py

```diff
@@ -322,6 +416,14 @@ def get_exploration_ids_matching_query(query_string, cursor=None):
     there are at least that many, otherwise it returns all remaining results.
     (If this behaviour does not occur, an error will be logged.) The method
     also returns a search cursor.
+
+    Args:
+        query_string: A search query string.
+        cursor: Search cursor.
+
+    Returns:
+        A list of exploration ids.
```

Comment:
A list of exploration ids matching the query.

(Generally, say the type and what the return value actually represents. In all cases, please make sure you read the code carefully to really understand what is going on; it is possible that existing docstrings are out-of-date or wrong. Part of the point of this PR is to try and learn the codebase, so feel free to do any research or additional delving that might be needed.)